tests/tensorizer_loader/test_tensorizer.py

```diff
@@ -240,7 +264,8 @@ def test_raise_value_error_on_invalid_load_format(vllm_runner):
             model_loader_extra_config=TensorizerConfig(tensorizer_uri="test"))
 
 
-def test_tensorizer_with_tp(vllm_runner):
+@pytest.mark.skip("Requires multiple GPUs")
```

Comment:
It would be better to use `@pytest.mark.skipif` in the case of the CI testing suite using (or ever will use) multiple GPUs. It feels hacky and incomplete for this test to skip unconditionally. Something like 

```python
@pytest.mark.skipif(not multiple_gpus_available())
```

would be better.

Reply:
I copied this check from some of the lora tests that need multiple GPUs ([example here](https://github.com/vllm-project/vllm/blob/4238bc82f24d5887784b04a353ed93e2360623b4/tests/lora/test_llama.py#L83)). I agree that a `skipif` would be better; however, with the current bug around launching a second distributed engine during the test run, only one of the tests can pass at a time anyways. If we can find a fix for that, then I think we should use the `skipif` as you suggest.